### DISCUSSION TITLE
Display pop up once per session?

### DISCUSSION THREAD

**Discussion by kontrabandmerch** [fancyapps/ui#365] [Q&A]
Hi There
We've been using the fancapps modal for years and it's worked really well for our online stores.

Recently a client has asked if we can display their mailing list form in a pop-up on page load.  Now I can get the pop-up to pop up with no problem, but I'd like to be able to not have it become an annoyance.

I've looked into controlling this with cookies, but everything I've found is years old and I can't get working with the current version.

Is there any way of being able to display the fancyapps modal once a week?

Sean

**Comment by fancyapps** [comment]
Hi,

All you have to do is to figure out how to execute any code snippet, in your case, once a week. 
If you will be able to, for example, execute `window.alert("hello");`, then simply replace that with `Fancybox.show(YOUR_CONTENT)`.
But, how to do that, is completely up to you. You can use cookies (server side or JS), local storage, etc.
